google/cloud/storage/client.py

```diff
@@ -836,6 +846,175 @@ def get_hmac_key_metadata(
         metadata.reload(timeout=timeout)  # raises NotFound for missing key
         return metadata
 
+    def generate_signed_post_policy_v4(
+        self,
+        bucket_name,
+        blob_name,
+        conditions,
+        expiration,
```

Comment:
Please put `expiration` before `conditions`.
This should also be changed in example.

Reply:
Oh, yeah, you mentioned that yesterday in fact, I just didn't completely got it, sorry